badger_api/requests.py

```diff
@@ -74,10 +74,13 @@ def fetch_all_claimable_balances(chain: str) -> Optional[Dict]:
 
 
 @lru_cache
-def fetch_token_names(chain: str) -> Optional[Union[Dict, List]]:
-    return http_client.get(f"{badger_api}/tokens?chain={chain}")
+def fetch_token_names(chain: str) -> Dict:
```

Comment:
Correct typing here will be `Optional[Union[Dict, List]]`. This is because function can either return None, or a valid deserialized JSON file

Reply:
I wonder if this is merge artifact?